app/models/scanned_book.rb

```diff
@@ -1,5 +1,8 @@
 # Generated via
 #  `rails generate worthwhile:work ScannedBook`
+
+require 'active_fedora/noid'
+
 class ScannedBook < ActiveFedora::Base
```

Comment:
We want every model to get a NOID (e.g. Pages, when we have them, and ArchivalWhatevers). I think maybe these overrides should be at a higher level or at least in a module that we can mix into future models without repeating this code, no?